spec/features/likes.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'User like', type: :feature do
+  before :each do
+    User.create(name: 'mike', email: 'mike@gmail.com', password: '111111')
+    Post.create(content: 'text', author_id: 1)
+  end
+
+  scenario 'has like' do
+    login_with 'mike@gmail.com', '111111'
+    click_on 'Like'
```

Comment:
**[REQUIRED]**
This line is causing this :point_down:  error.
```
Failure/Error: click_on 'Like'
     
     Capybara::ElementNotFound:
       Unable to find link or button "Like"
```

Reply:
We have also resolved this.